data-prepper-plugins/mutate-string-processors/src/main/java/org/opensearch/dataprepper/plugins/processor/mutatestring/SubstituteStringProcessorConfig.java

```diff
@@ -27,15 +29,22 @@ public static class Entry {
                 "be escaped using <code>\\\\</code> when using double quotes and <code>\\</code> when using single quotes. " +
                 "See <a href=\"https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/regex/Pattern.html\">Java Patterns</a> " +
                 "for more information.")
+        @ExampleValues({
+                @Example(value = "/+", description = "Matches at least one forward slash and can be used to normalize a path."),
+                @Example(value = "'[?&#=]'", description = "Matches some special characters common in URIs.")
```

Comment:
Single quotes not necessary?
```suggestion
                @Example(value = "[?&#=]", description = "Matches some special characters common in URIs.")
```